new/LIT7103PrayOTprophets.xml

```diff
@@ -0,0 +1,112 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7103PrayOTprophets" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Collection of prayers of the prophets of the Old Testament (ወሶበ፡ ጸርሐ፡ ስመከ፡ ...)</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BLorient12859"/>
+                    <witness corresp="BNFabb107"/>
+                    <witness corresp="BNFabb270"/>
+                    <witness corresp="BNFeth320"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Collection of prayers of the prophets of the Old Testament, each of which are introduced with 
+                    <foreign xml:lang="gez">ወሶበ፡ ጸርሐ፡</foreign> (engl. "When he cried") or 
+                    <foreign xml:lang="gez">ወሶበ፡ ጸርሑ፡</foreign> (engl. "When they cried") and is often continued with 
+                    <foreign xml:lang="gez">ስመከ፡</foreign> (engl. "your name"), possibly related to 
+                    <ref type="work" corresp="LIT1828Mahale"/>, <ref type="work" corresp="LIT2277salotu"/>,
+                    <ref type="work" corresp="LIT2265salota"/> or <ref type="work" corresp="LIT2250salota"/>.</p>
```

Comment:
this is not quite correct. the entire collection may be related to LIT1828Mahale and the single prayers may be related to, or renarrations of, the songs of the prophets otherwise contained in LIT1828Mahale, so these links must be attached to the parts in question.

Reply:
listRelation could also be helpful

Reply:
OK. Shall I use dc:relation or is there a more specific relation for this case?

Reply:
dc:relation should be OK, thank you